iroh/src/client/blobs.rs

```diff
@@ -1248,4 +1248,91 @@ mod tests {
 
         Ok(())
     }
+
+    /// Download a blob from oneself
+    #[tokio::test]
+    async fn test_blob_get_self() -> Result<()> {
```

Comment:
FWIW, instead of all of the `.context()` calls, you can also just use `TestResult` instead of `anyhow::Result`.
This way you'll get the exact line that fails in the error that's thrown.
(It essentially behaves the same as if you did `.unwrap()` everywhere)

Reply:
Changed the new tests to that. Could also do a big pass over everything, but not today..